src/NServiceBus.Testing.Tests/Sagas/SagaWithSagaFinder.cs

```diff
@@ -0,0 +1,160 @@
+﻿namespace NServiceBus.Testing.Tests.Sagas
+{
+    using System;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class SagaWithSagaFinder
+    {
+        [Test]
+        public async Task TestSagaWithPropertyNameAndValueSagaFinder()
+        {
+            var testableSaga = new TestableSaga<ShippingPolicy, ShippingPolicyData>();
+            testableSaga.MockSagaFinder<OrderBilled>(message => message.OrderId, message => ("OrderId", message.OrderId));
```

Comment:
These examples are confusing to me, because if OrderBilled really had an OrderId then we'd just just map the correlation id and be done. Given that we link to these at the [end of the documentation page](https://docs.particular.net/nservicebus/testing/saga-scenario-testing#additional-examples) they need to be more realistic. And maybe that would help me to understand too.